app/src/org/commcare/preferences/DeveloperPreferences.java

```diff
@@ -315,4 +317,8 @@ public static String formLoadPayloadStatus() {
     public static boolean isDetailTabSwipeActionEnabled() {
         return doesPropertyMatch(DETAIL_TAB_SWIPE_ACTION_ENABLED, CommCarePreferences.YES, CommCarePreferences.YES);
     }
+
+    public static boolean isOfflineUpdateEnabled() {
+        return doesPropertyMatch(OFFLINE_UPDATE_ENABLED, CommCarePreferences.YES, CommCarePreferences.YES);
```

Comment:
What's the expected reason it might be disabled? It's actually a pretty sketchy attack vector in its current form, because someone could use the app manager to get to the update screen, then "replace" someone's app install with a malicious version (moving the submit/sync server to their Man in the Middle passthrough) without needing to be logged in, but I think if we're worried about that we shouldn't turn it on by default.